_includes/domain-search.html

```diff
@@ -20,7 +20,7 @@ <h2>
                     Read more about choosing your .gov domain.
                 </a>
             </h3>
-            <form class="usa-search usa-search--domain" role="search">
+            <form class="usa-search usa-search--domain-secondary" role="search">
```

Comment:
Renamed class name given this variant (light blue) will only be when a domain search bar is on a blue background which will be less common than our new gray variant for white backgrounds